userspace/libsinsp/sinsp.cpp

```diff
@@ -323,6 +323,8 @@ void sinsp::init()
 		m_num_cpus = 0;
 	}
 
+	m_boot_ts_epoch = sinsp_utils::get_host_boot_time_ns();
```

Comment:
The bad news is that this won't work in all possible sinsp use cases, especially when reading scap files on platforms including Linux/Win/Mac as in those cases the final value that we compute (when doing the difference) would be referring to the client that reads the scap file instead of the original host, then creating too many alerts! :scream: 

The good news is that it's pretty easy to move it to the appropriate place! If you take a look at the structure `scap_machine_info` you can easily rename `reserved1` in `boot_ts_epoch`, then look for all the places where `m_machine_info.reserved1 = 0;` is set and change it appropriately by calling the `get_host_boot_time_ns` function which needs to be moved to scap (at least for open live, udig and nodriver, you can leave `0` in the plugin one with a comment). You will then have access to that field in the inspector through the `m_machine_info` field!

Reply:
Great catch @LucaGuerra, thanks for that. In addition, it seems more elegant this way. Please let me should you prefer additional changes. @Andreagit97 also feel free to directly address them given we are different timezones.